Doc/library/shutil.rst

```diff
@@ -226,16 +226,19 @@ Directory and files operations
 
 
 .. function:: copytree(src, dst, symlinks=False, ignore=None, \
-              copy_function=copy2, ignore_dangling_symlinks=False, \
-              dirs_exist_ok=False)
+              copy_stat=True, copy_function=copy2, \
+              ignore_dangling_symlinks=False, dirs_exist_ok=False)
```

Comment:
Inserting this new parameter before existing ones will break existing uses if passed positionally. Was this position chosen for a reason?

Reply:
The position was chosen so it'd be near the `copy_function` parameter.